build.gradle

```
@@ -41,7 +41,7 @@ ext {
   joptVersion = '4.9'
 
   slf4jVersion = '1.7.30'
-  log4jVersion = '2.15.0'
+  log4jVersion = '2.17.0'
```

there's been a CVE patch release since then (again 😅)
```suggestion
  log4jVersion = '2.17.1'
```